templates/feeds-index.item

```diff
@@ -1,10 +1,12 @@
       <div class="row mb-4">
-        <div class="col-md-2 d-flex justify-content-center">
-          <img src="{IMAGE_URL}" alt="{TITLE}" class="img-fluid" />
+        <div class="col-md-2 d-flex justify-content-center align-items-center">
+          <a href="https://smotrim.ru/podcast/{FEED_ID}" target="_blank" style="width: 104px; height: 104px;">
```

Comment:
Do we really need size duplication for `<a>` in the case when nested image already have size?

Reply:
without size a is higher than img

Reply:
it is unnecessary here due to `align-items-center`. But without it `a` is higher.

Reply:
Why it is higher? Or wider?

Reply:
Due to `align-items: stretch` set as default value.

Reply:
moving `img` into `a` does not help. HTML ignores any whitespaces between tags. [as i understand:)](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace)